app/models/order.rb

```diff
@@ -391,8 +383,21 @@ def distribute_transport
 
   # Updates the "price" attribute of GroupOrders or GroupOrderResults
   # This will be either the maximum value of a current order or the actual order value of a finished order.
-  def update_price_of_group_orders
-    group_orders.each { |group_order| group_order.update_price! }
+  def update_price_of_group_orders!
+    group_orders.each(&:update_price!)
```

Comment:
i think this is more a method, which should be optimized, since it happens often :wink:

Reply:
Yes, good one.
There are quite a few things that can be optimized here, but this one is indeed called often.